dotnet/api/Presentation/Cli/Command.cs

```diff
@@ -75,8 +75,8 @@ protected GBApiContext _testDatabaseContext
             {
                 if (_cachedTestDatabaseContext == null)
                 {
-                    var connection = _configurationRoot.GetTestDatabaseConnection();
-                    _cachedTestDatabaseContext = new GBApiContext(connection, null);
+                    var connectionStringBuilder = _configurationRoot.GetTestDatabaseConnectionStringBuilder();
+                    _cachedTestDatabaseContext = new GBApiContext(connectionStringBuilder.ConnectionString, null);
```

Comment:
Suggestion: Consider alternative constructor for GBApiContext or using named parameters to describe the `null`